tests/resources/modules/d8/woot/woot.services.yml

```diff
@@ -0,0 +1,14 @@
+parameters:
+  woot.commands:
+    - woot.manager
+    - woot.command
+    - greet.command
+services:
+  woot.manager:
+    class: Drupal\woot\WootManager
+    arguments: ['@current_user']
+  woot.command:
+    class: Drupal\woot\Command\WootCommands
```

Comment:
Yes, good point. The 'woot.commands' parameter adds to this confusion; fortunately, that particular entry will go away with tagged services. I'll clean up the service name after that.